cmd/orchestrator/main.go

```diff
@@ -122,6 +122,7 @@ func main() {
 	// routes
 	r.Get("/healthz", s.HealthzHandler)
 	r.Post("/campaign", s.CampaignHandler)
+	r.Post("/cancel", s.CancelHandler)
```

Comment:
I'd like to see the campaign related endpoints bundled under `/campaign`, however, maybe that's a separate ticket to clean this up.